lambdas/indexer/.gitignore

```diff
@@ -3,4 +3,5 @@
 /.chalice/config.json
 /.chalice/deployed/*.json
 /lambda-policy.json
-/vendor/azul_changes.py
+/vendor/*
```

Comment:
Next time you push, please push a history with two commits. First commit should have the changes to these `.gitignore` files. Second commit is everything else.

Reply:
This hasn't been addressed.

Reply:
@hannes-ucsc This change was made when we try to streamline `make deploy` to install additional wheels (b7690914). Since then, we identified that the compatibility between two Python versions (3.6 and 3.7) caused the issue and we addressed that by ensuring that everyone must use Python 3.6 for the deployment.

I think this change seems to be unnecessary at this point and I think of reverting this change. What do you think?